src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,115 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.command.Command;
+import duke.command.DeadlineCommand;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.EventCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.ToDoCommand;
+import duke.exception.DukeException;
+import duke.exception.DukeInsufficientParametersException;
+import duke.exception.DukeInvalidDateException;
+import duke.exception.DukeInvalidParametersException;
+import duke.exception.DukeMissingFlagException;
+import duke.exception.DukeUnknownCommandException;
+
+public class Parser {
+    private static String[] acceptedCommands = {
```

Comment:
perhaps you could try using enum. Or since they are constants, let acceptedCommands be in caps like ACCEPTED_COMMANDS?